packaging/generic-deb/debian/dirs

```diff
@@ -1,3 +1,4 @@
 etc/ecs
 var/log/ecs
 var/lib/ecs
+var/cache/ecs
```

Comment:
is the data file also suppose to be included?

Reply:
yes i will add